plugins/cache-portals-on-map.js

```diff
@@ -30,10 +30,36 @@ window.plugin.cachePortalDetailsOnMap.entityInject = function(data) {
   data.callback(ents);
 };
 
+// extend portal's GUID lookup to search in cached portals as well
+window.cachePortalDetailsOnMap.findPortalGuidByCacheE6 = function(latE6, lngE6) {
+  let guid = window.cachePortalDetailsOnMap.findPortalGuidByPositionE6old(latE6, lngE6);
+
+  // PLUGIN: cache > cache
+  if (guid == null) {
+    const cache = window.plugin.cachePortalDetailsOnMap;
+    if (cache && cache.cache) {
+      for (let g in cache.cache) {
+        let p = cache.cache[g];
+        if (latE6 == p.ent[2][2] && lngE6 == p.ent[2][3]) {
+//        console.log("newFindPortalGuidByPositionE6 matched in cache! %s %s -> [%s] = %o", latE6, lngE6, g, p);
+          guid = g;
+          break;
+        }
+      }
+    }
+  }
+  return guid;
+};
```

Comment:
This improvement has a very little sense, because "Cache viewed portals on map" is extremely limited and never keeps much data in it's cache. Also, `findPortalGuidByPositionE6` hijacking looks like hack: juist imaging - there are other caching plugins in the wild, and what if every of them would hijack that single function.

If we really need this functionality - than we should better implement via some clean hook.